src/test/java/FishingExample.java

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Do not put the test mod in the default package. Please follow the existing examples on how to structure the package.